mods/_testmod/mod.lua

```diff
@@ -19,6 +19,13 @@ function Mod:init()
 end
 
 function Mod:preInit()
+    -- Set the game title to something different
+    Game:setGameTitle("Example Mod")
+
+    -- Set the windows icon to something different
+    -- For mods please use Assets.getTextureData has argument
+    Game:setGameIcon( Assets.getTextureData( "kristal/banana_1" ) )
```

Comment:
This isn't where `Assets.getTextureData` should be, it should be handled in the function itself